Use a forced metric type of gauge for ccmRejectedPhones and ccmUnregisteredPhones.

jmw51798 opened the pull request:
### What does this PR do?
<!-- A brief description of the change being made with this pull request. -->
Use a forced metric type of gauge for ccmRejectedPhones and ccmUnregisteredPhones in the _cisco-voice.yaml default profile.  Without this those two metrics are incorrectly inferred to be rates. 

### Motivation
<!-- What inspired you to submit this pull request? -->
A customer reported the issue.

### Additional Notes
<!-- Anything else we should know when reviewing? -->
To test:
1) ddev env start snmp py3.11-false -e DD_TAGS="ndm_qa:test-17722" -e DD_SITE=datad0g.com -e DD_DD_URL=https://app.datad0g.com -e DD_API_KEY=$DATADOG_STAGING_ORG2_API_KEY
2) ddev env config edit snmp py3.11-false
Change community_string from public to cisco_icm
3) ddev env reload snmp py3.11-false
4) Go to https://dd.datad0g.com/metric/explorer and confirm that ccmRejectedPhones and ccmUnregisteredPhones metrics show up as non-zero.

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration, e2e)
- [ ] [Changelog entries](https://datadoghq.dev/integrations-core/guidelines/pr/#changelog-entries) must be created for modifications to shipped code
- [ ] Add the `qa/skip-qa` label if the PR doesn't need to be tested during QA.
- [ ] If you need to backport this PR to another branch, you can add the `backport/<branch-name>` label to the PR and it will automatically open a backport PR once this one is merged

NouhaManai96 reviewed (commented):
lgtm once checks pass
you need to update the tests to reflect the change in the metric type, you can just [include the metrics in the `ccm_gauges` instead of `ccm_counts`](https://github.com/DataDog/integrations-core/blob/master/snmp/tests/test_profiles.py#L185-L187)

jmw51798 commented:
> lgtm once checks pass you need to update the tests to reflect the change in the metric type, you can just [include the metrics in the `ccm_gauges` instead of `ccm_counts`](https://github.com/DataDog/integrations-core/blob/master/snmp/tests/test_profiles.py#L185-L187)

Thanks @NouhaManai96, I fixed the tests and added a changelog.  Looks like all of the checks are passing now.
